pdc_client/__init__.py

```diff
@@ -187,10 +189,10 @@ def obtain_token(self):
 
     def get_paged(self, res, **kwargs):
         """
-        This call is equivalent to ``res(**kwargs)``, only it retrieves all pages
-        and returns the results joined into a single iterable. The advantage over
-        retrieving everything at once is that the result can be consumed
-        immediately.
+        This call is equivalent to ``res(**kwargs)``, if there is no self.page
```

Comment:
@hluk and @simozhan I think we can keep this page parameter in the init function in the future.
But when we call PDCClient in new codes, we don't use it. It is just for old codes. 
By default we just consider people use PDCClient to get all data. So below codes make sence to me.
`class PDCClient(object):
     def __init__(self, server,  page_size=None):

client = PDCClient(server)
`